README.md

```diff
@@ -10,42 +25,47 @@
 ![Commit activity](https://img.shields.io/github/commit-activity/w/lyft/flytectl.svg?style=plastic)
 ![Commit since last release](https://img.shields.io/github/commits-since/lyft/flytectl/latest.svg?style=plastic)
 
-Flytectl is designed to be a portable, lightweight, CLI for working with Flyte.  It is written in Golang and can access FlyteAdmin
+FlyteCTL is designed to be a portable and lightweight CLI to work with Flyte. It is written in Golang and can access [FlyteAdmin](https://github.com/flyteorg/flyteadmin/)—the control plane of Flyte.
 
-## Docs
+## 🚀 Quick Start
 
-Docs are generated using Sphinx and are available at [flytectl.rtfd.io](https://flytectl.rtfd.io)
-Generating docs locally can be accomplished by running make gendocs from within the docs folder
+* Install FlyteCTL with bash or shell script
 
+    * Bash
+        ```bash
+        $ brew install flyteorg/homebrew-tap/flytectl
 
-## Installation
+        # Upgrade flytectl 
+        $ brew upgrade flytectl
```

Comment:
Can we remove the brew upgrade and add a common section for an upgrade that will use `flytectl upgrade`